Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyQuestions.kt

```diff
@@ -88,54 +88,27 @@ fun Question(
     onAnswer: (Answer<*>) -> Unit,
     onAction: (Int, SurveyActionType) -> Unit,
     onDoNotAskForPermissions: () -> Unit,
-    openSettings: () -> Unit,
```

Comment:
Why was this removed? feels like something outside the scope of an alpha update?

Reply:
new permissions api. nevermind